core/controllers/moderator.py

```diff
@@ -73,12 +73,8 @@ def get(self):
     @acl_decorators.can_access_moderator_page
     def post(self):
         """Handles POST requests."""
-        featured_activity_reference_dicts = self.normalized_payload.get(
+        featured_activity_references = self.normalized_payload.get(
             'featured_activity_reference_dicts')
-        featured_activity_references = [
```

Comment:
Not required, since after schema validation the handlers will receive an objects instead of dicts.